tests/usecase/test_emit_metrics.py

```diff
@@ -79,3 +108,54 @@ def test_posting_metrics_works(self):
 
         time.sleep(10)
         self.assert_string_not_in_recent_logs("MENDIX-METRICS: ")
+
+
+class TestFreeAppsMetricsEmitter(TestCase):
+    def setUp(self):
+        interval = 10
+        m2ee = Mock()
+
+        self.metrics_emitter = FreeAppsMetricsEmitterThread(interval, m2ee)
+        self.metrics_emitter._get_munin_stats = Mock(
+            return_value=copy.deepcopy(mock_user_session_metrics)
+        )
+        self.metrics_emitter.emit = Mock()
+        self.metrics_emitter.setDaemon(True)
+
+    def test_inject_user_session_metrics(self):
+        stats = {"key": "value"}
+        expected_stats = copy.deepcopy(stats)
+        expected_stats["mendix_runtime"] = mock_user_session_metrics
+
+        stats = self.metrics_emitter._inject_user_session_metrics(stats)
+        self.assertEqual(True, self.metrics_emitter._get_munin_stats.called)
+        self.assertEqual(expected_stats, stats)
+        print(stats)
+
+    def test_inject_user_session_metrics_when_mendix_runtime_metrics_already_present(
+        self
+    ):
+        stats = {
+            "key": "value",
+            "mendix_runtime": {"other_key": "other_value"},
+        }
+        expected_stats = copy.deepcopy(stats)
+        expected_stats["mendix_runtime"].update(mock_user_session_metrics)
+
+        stats = self.metrics_emitter._inject_user_session_metrics(stats)
+        self.assertEqual(True, self.metrics_emitter._get_munin_stats.called)
```

Comment:
```suggestion
        self.assertTrue(self.metrics_emitter._get_munin_stats.called)
```

Please use `self.assertTrue` instead.

Reply:
Changed!